### PR TITLE
fix(stop): mobile stop size decreased

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

**Fixes:** #11

## Summary:

This change adds a media query to decrease the size of the stop on mobile to 12px. This is achieved using a media query with max-width at `auro-breakpoint-sm` (660px) and then setting changing the height and width of the leg from `--auro-size-md` to `--auro-size-sm`.

An additional change is that I noted that when the previous PR was merged in, the API.md file had not been updated for the new `canceled` parameter. This PR updates the API.md file to include it.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other style change only)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/style-flight-segment.scss [hunk 1]
```diff
@@ -46,6 +48,11 @@
   &--layover {
     background-color: var(--auro-color-ui-active-on-light);
   }
+
+  @include auro_breakpoint($max: $auro-breakpoint-sm) {
```
Comment (geoffrich) [inline src/style-flight-segment.scss:52]: We've written breakpoints the other way around in this repo, e.g. https://github.com/AlaskaAirlines/auro-flightline/blob/4e7d755759a36b5b8c72babb2e1f7e73cb1b991b/src/style-flightline.scss#L54-L58

Can we set width/height to auro-size-sm at first and then override them in the breakpoint for larger screen sizes?

Comment (jason-capsule42) [reply src/style-flight-segment.scss:52]: Yeah, that makes sense. I'll switch it around.

Comment (jason-capsule42) [reply src/style-flight-segment.scss:52]: Yeah, that makes sense. I'll switch it around.

**Review by geoffrich — APPROVED** [review]

**Pull request merged**